apps/mobile/CashManager/.gitignore

```diff
@@ -0,0 +1,14 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
```

Comment:
You should add all those ignored files/folders to the .gitignore file a the root of the repository for a better visibility :)

and remove the `/` character at the start of the path or udpate it wieh `/apps/mobile/...`

Reply:
Done